spec/factories/kits.rb

```diff
@@ -17,7 +17,9 @@
     organization
 
     after(:build) do |instance, _|
-      instance.line_items << create(:line_item)
+      if instance.line_items.blank?
+        instance.line_items << create(:line_item)
```

Comment:
Since we're no longer automatically using `@organization`, we should explicitly pass `organization: instance.organization` here.